### PR TITLE
Update youtube admin list

### PR DESCRIPTION
**Pull request by mrbobbytables**
Just cleaning up the list of moderators. I added a section for emergency or backup admins that have access, but don't really participate on a regular basis.

/hold

/assign @kaslin @palnabarun

### CODE REVIEW

File: communication/moderators.md [hunk 1]
```diff
@@ -142,32 +142,26 @@ Email: contributors@kubernetes.io
...
 | Name                | Kubernetes Slack ID   | Region   | Timezone                                                |
 | ------------------- | --------------------- | -------- | ------------------------------------------------------- |
-| Alison Dowdney      | @alisondy             | EMEA     | [GMT - Greenwich Mean Time](https://time.is/GMT)        |
-| Jeffrey Sica        | @jeefy                | Americas | [ET - Eastern Time (US East Coast)](https://time.is/ET) |
-| Bob Killen          | @mrbobbytables        | Americas | [ET - Eastern Time (US East Coast)](https://time.is/ET) |
 | Chris Short         | @chrisshort           | Americas | [ET - Eastern Time (US East Coast)](https://time.is/ET) |
-| Taylor Dolezal      | @onlydole             | Americas | [PT - Pacific Time (US West Coast)](https://time.is/PT) |
-| David McKay         | @rawkode              | EMEA     | [GMT - Greenwich Mean Time](https://time.is/GMT)        |
-| Marky Jackson       | @markyjackson-taulia  | Americas | [PT - Pacific Time (US West Coast)](https://time.is/PT) |
 | Nigel Brown         | @nigel                | Americas | [CT - Central Time (US Central)](https://time.is/CT)    |
```
Comment (SergeyKanzhelev) [inline communication/moderators.md:150]: Cannot find nigel on Slack:

image

Comment (SergeyKanzhelev) [reply communication/moderators.md:150]: oh, it says there are multiple options

Comment (SergeyKanzhelev) [reply communication/moderators.md:150]: sorry for the noise. Figured it out

**Comment by jeefy** [comment]
LGTM

**Pull request merged**